.env.example

```diff
@@ -1,8 +1,20 @@
 MONGODB_USERNAME=root
 MONGODB_PASSWORD=password
-MONGODB_HOST_TYPE=local
+MONGODB_HOST_TYPE=local # local | atlas
 MONGODB_HOST="localhost:27017"
+SERVER_PORT=8081
+K1_ACCESS_TOKEN=feedkray
+IS_CLUSTER_ZERO=`
```

Comment:
@D-B-Hawk can you confirm this line and the set like it below - i'm not familiar with a single tick convention, feels like a bad find/replace

Reply:
mistakes were made 🙃